cmd/janitor/janitor.go

```diff
@@ -44,6 +45,7 @@ var (
 
 func init() {
 	flag.Var(&rTypes, "resource-type", "comma-separated list of resources need to be cleaned up")
+	flag.StringVar(&rTypesConfig, "resource-type-from-config", "", "extract a list of resources need to be cleaned up from boskos' config file")
```

Comment:
could you call out that --resource-type and --resource-type-from-config can co-exist?

Reply:
Done. Thanks. Added the priority order too